src/Model/GroundWaterFlow/gwf3lak8.f90

```diff
@@ -5098,6 +5098,14 @@ subroutine lak_process_obsID(obsrv, dis, inunitobs, iout)
       if (obsrv%ObsTypeId=='LAK' .or. obsrv%ObsTypeId=='CONDUCTANCE' .or. &
           obsrv%ObsTypeId=='WETTED-AREA') then
         call extract_idnum_or_bndname(strng, icol, istart, istop, nn2, bndname)
+        if (len_trim(bndName) < 1 .and. nn2 < 0) then
```

Comment:
@emorway-usgs, thanks for this!  By any chance, can you quickly check MAW and see if the same situation exists there?

Reply:
@langevin-usgs Yes, I'll take a look at MAW and the other advanced packages.

Reply:
@langevin-usgs Attempting to bring closure to this review comment with #715